src/Common/ErrorCodes.cpp

```diff
@@ -538,7 +538,8 @@
     M(999, KEEPER_EXCEPTION) \
     M(1000, POCO_EXCEPTION) \
     M(1001, STD_EXCEPTION) \
-    M(1002, UNKNOWN_EXCEPTION)
+    M(1002, UNKNOWN_EXCEPTION) \
+    M(1003, INVALID_SHARD_ID)
```

Comment:
BTW it is better to use values < 999 for such exceptions (since error codes >= 999 is something specific)